inc/saifdb.h

```diff
@@ -190,6 +209,15 @@ typedef enum _sai_fdb_entry_attr_t
      */
     SAI_FDB_ENTRY_ATTR_ALLOW_MAC_MOVE,
 
+    /**
+     * @brief Specify FDB entry state
+     *
+     * @type sai_fdb_entry_state_t
+     * @flags CREATE_AND_SET
```

Comment:
Should it be `READ_ONLY` ?
Still not sure about a real use case for this attribute.